app/src/main/java/com/example/todolistproject/model/Task.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.todolistproject.model
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "Task")
+data class Task(
+
+    @PrimaryKey(autoGenerate = true)
+    var id: Long?,
+
+    @ColumnInfo(name = "isCompleted")
+    var isCompleted: Boolean? = false,
+
+    @ColumnInfo(name = "title")
+    var title: String,
+
+    @ColumnInfo(name = "description")
+    var description: String? = "",
+
+    @ColumnInfo(name = "startDate")
+    var startDate: String?,
+
+    @ColumnInfo(name = "endDate")
+    var endDate: String?,
+) {
+    constructor(): this(null, false, "", "", "", "")
+}
```

Comment:
```suggestion
)
```
data class 에서 장점을 살릴려면 kotlin기본값을 사용하시면 더 좋아요